### PR TITLE
URI to URL to address #11450

### PR DESCRIPTION
**Pull request by Coder206**
@asajeffrey Please note, there is an error and a warning: 

Compiling script v0.0.1 (file:///E:/files/dev/github/servo/components/script)
E:\files\dev\github\servo\components\script\dom\xmlhttprequest.rs:1044:5: 1048:6 warning: method is never used: `insert_trusted_header`, #[warn(dead_code)] on by default
E:\files\dev\github\servo\components\script\dom\xmlhttprequest.rs:1044     fn insert_trusted_header(&self, name: String, value: String) {

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11765)

### CODE REVIEW

File: components/script/dom/htmliframeelement.rs [hunk 1]
```diff
@@ -67,7 +67,7 @@ pub struct HTMLIFrameElement {
     pipeline_id: Cell<Option<PipelineId>>,
     subpage_id: Cell<Option<SubpageId>>,
     sandbox: MutNullableHeap<JS<DOMTokenList>>,
-    sandbox_allowance: Cell<Option<u8>>,
+    sandbox_allowance: Cell<Option<SandboxAllowance>>,
```
Comment (KiChjang) [inline components/script/dom/htmliframeelement.rs]: I don't understand this change.

**Comment by KiChjang** [comment]
This fails building in Travis CI.

File: components/script/dom/htmliframeelement.rs (hunk 1 above)
Comment (KiChjang) [inline components/script/dom/htmliframeelement.rs]: Ah I see, this is intended to fix #11599. Please build your branch and fix the compiler errors that result from this change.

**Comment by Coder206** [comment]
#11789

**Pull request closed without merging**